cmd/ipfs/main_test.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
-	"testing"
-
 	"github.com/ipfs/go-ipfs/commands"
+
+	gc "gx/ipfs/QmPp8w6dCzx6vPNre8QVSbN7GCY9M7ornWesnrJg48vYxE/check.v1"
 )
 
-func TestIsCientErr(t *testing.T) {
-	t.Log("Catch both pointers and values")
-	if !isClientError(commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified value")
-	}
-	if !isClientError(&commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified pointer")
-	}
+func (m mainRun) TestIsCientErr(c *gc.C) {
```

Comment:
I disagree completely.
I think more mantainers of this project should take a look and give more feedback.
The c.Assert besides of line number and fail message it contains  these lines as well "expected: value, obtained: value". which it gives us way more information on "what went wrong". If you want I can use testify to write these tests.
But keep relaying and using the testing pkg from std lib it's not good idea, in the long run. (in my opinion).